<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginPage</class>
 <widget class="QWidget" name="LoginPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1093</width>
    <height>613</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LayoutDirection::LeftToRight</enum>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="10" column="1">
    <widget class="QPushButton" name="loginButton">
     <property name="styleSheet">
      <string notr="true">QPushButton {
    background-color: rgb(28, 130, 255);
    color: white;
    border: none;
    border-radius: 15px; 
    padding: 10px 20px; 
    font-size: 15px; 
    font-weight: bold; 
}

QPushButton:hover {
    background-color: rgb(20, 100, 200);
    color: white;
}

QPushButton:pressed {
    background-color: rgb(15, 80, 160); 
    border: 2px solid rgb(241, 92, 34);
    color: red; 
}
</string>
     </property>
     <property name="text">
      <string>Sign In</string>
     </property>
    </widget>
   </item>
   <item row="9" column="1">
    <spacer name="verticalSpacer_3">
     <property name="orientation">
      <enum>Qt::Orientation::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Policy::Preferred</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="13" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="label">
         <property name="styleSheet">
          <string notr="true">QLabel {
    color: rgb(90, 90, 100);
    font: bold 12px &quot;Arial&quot;;
}
</string>
         </property>
         <property name="text">
          <string>Don't have an account?</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="toCreateAccountButton">
         <property name="styleSheet">
          <string notr="true">QPushButton {
    border: none;
    background-color: transparent;
    color: black;
    font: bold 14px &quot;Arial&quot;;
}

QPushButton:hover {
    color: rgb(0, 102, 204);
}</string>
         </property>
         <property name="text">
          <string>Sign Up</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="5" column="3">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="7" column="1">
    <widget class="QLabel" name="titleLabel_3">
     <property name="styleSheet">
      <string notr="true">QLabel {
    color: rgb(28, 130, 255);
    font: bold 14px &quot;Arial&quot;;
}

QLabel:hover {
    color: rgb(20, 90, 200);
}
</string>
     </property>
     <property name="text">
      <string>Password</string>
     </property>
    </widget>
   </item>
   <item row="15" column="1">
    <spacer name="verticalSpacer_4">
     <property name="orientation">
      <enum>Qt::Orientation::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="1">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Orientation::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="8" column="1">
    <widget class="QLineEdit" name="loginPasswordEdit">
     <property name="styleSheet">
      <string notr="true">QLineEdit {
    border: none;
    border-bottom: 3px solid rgb(28, 130, 255);
    background-color: transparent;
    color: black;
    padding: 5px;
    font: bold 16px &quot;Arial&quot;;
}

QLineEdit:focus {
    border-bottom: 3px solid rgb(20, 90, 200);
    outline: none;
}

</string>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::EchoMode::Password</enum>
     </property>
     <property name="placeholderText">
      <string>Enter Password</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <spacer name="verticalSpacer_5">
     <property name="orientation">
      <enum>Qt::Orientation::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="6" column="1">
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Orientation::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Policy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="4" column="1">
    <widget class="QLabel" name="titleLabel_2">
     <property name="styleSheet">
      <string notr="true">QLabel {
    color: rgb(28, 130, 255);
    font: bold 14px &quot;Arial&quot;;
}

QLabel:hover {
    color: rgb(20, 90, 200);
}
</string>
     </property>
     <property name="text">
      <string>User Name</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="5" column="1">
    <widget class="QLineEdit" name="loginUsernameEdit">
     <property name="styleSheet">
      <string notr="true">QLineEdit {
    border: none;
    border-bottom: 3px solid rgb(28, 130, 255);
    background-color: transparent;
    color: black;
    padding: 5px;
    font: bold 16px &quot;Arial&quot;;
}

QLineEdit:focus {
    border-bottom: 3px solid rgb(20, 90, 200);
    outline: none;
}

</string>
     </property>
     <property name="placeholderText">
      <string>Enter User Name</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLabel" name="titleLabel">
     <property name="styleSheet">
      <string notr="true">QLabel {
    color: black; 

    font: bold 35px &quot;Palatino Linotype&quot;;
    text-align: center;
}
</string>
     </property>
     <property name="text">
      <string>Sign In</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
